src/LanguageServer/Protocol/Features/Options/SolutionCrawlerOptionsStorage.cs

```diff
@@ -17,21 +17,12 @@ internal static class SolutionCrawlerOptionsStorage
     public static readonly PerLanguageOption2<BackgroundAnalysisScope> BackgroundAnalysisScopeOption = new(
         "dotnet_analyzer_diagnostics_scope", defaultValue: BackgroundAnalysisScope.Default, group: s_backgroundAnalysisOptionGroup, serializer: EditorConfigValueSerializer.CreateSerializerForEnum<BackgroundAnalysisScope>());
 
-    /// <summary>
-    /// Option to turn configure background analysis scope for the current solution.
-    /// </summary>
-    public static readonly Option2<BackgroundAnalysisScope?> SolutionBackgroundAnalysisScopeOption = new(
-        "SolutionCrawlerOptionsStorage_SolutionBackgroundAnalysisScopeOption", defaultValue: null, group: s_backgroundAnalysisOptionGroup, serializer: EditorConfigValueSerializer.CreateSerializerForNullableEnum<BackgroundAnalysisScope>());
```

Comment:
>SolutionCrawlerOptionsStorage_SolutionBackgroundAnalysisScopeOption [](http://example.com/codeflow?start=9&length=67)

Is the class name / namespace still appropriate?

Reply:
probably not.  but i'm ok not touching that.